src/backend/TrafficCourts/TrafficCourts.Arc.Dispute.Service/Configuration/SftpConfig.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace TrafficCourts.Arc.Dispute.Service.Configuration
+{
+    public class SftpConfig
+    {
+        public const string Section = "Sftp";
+        public string Host { get; set; }
```

Comment:
I've seen these warnings elsewhere in the existing code and have been trying to clean them up. I'm not sure on .NET best practices, but from my reading it would be good to address the warnings (or we fix the editorconfig to ignore these).
I think this means that the Host property is defined here as not null.  Yet there is no constructor that specifically sets the value.  So it possible to do a 
`SftpConfig sftpConfig= new SftpConfig();`
and have `SftpConfig.Host` be null.

Either define this property as nullable, or add a constructor and specifically set the value to something.
ie.
`public string? Host { get; set; }`

Reply:
Declared Host, Username and Password properties as nullable